File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.js
function solution(s) {
    
    let word = s.split(' ')
    
    let answer = []
    
    
    for ( i = 0; i < word.length; i++ ) { 
        
        let newWord = ''
                                        
        for ( j = 0; j < word[i].length; j++ ) {                     
            
            if ( j === 0 || j % 2 === 0) {                
                                           
                newWord += word[i][j].toUpperCase()               
                                
            } else {
                
                newWord += word[i][j].toLowerCase()
            }                  
        }              

        answer.push(newWord)
         
    }
        
    
    return answer.join(' ');
}

File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.py
def solution(s):
    answer = []
    words = s.split(' ')
    
    for word in words:
        new = ''        
        for i in range(len(word)):
            new += word[i].upper() if i % 2 == 0 else word[i].lower()
        answer.append(new)
            
    return " ".join(answer)